src/libsyntax/ast.rs

```diff
@@ -399,6 +399,16 @@ impl Generics {
     pub fn is_parameterized(&self) -> bool {
         self.is_lt_parameterized() || self.is_type_parameterized()
     }
+    pub fn span_for_name(&self, name: &str) -> Option<Span> {
```

Comment:
I remember doing such a thing but can't find the PR. From my point of view, this is correct. However I can totally overpass a better solution.

Anything in mind @eddyb?